### PR TITLE
Graphql

### PR DESCRIPTION
**Pull request by ejdelsztejn**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)

Correct, complete survey product endpoint

### CODE REVIEW

**Review by AurumValian — APPROVED** [review]
Loving the objects! Thanks for setting everything up well! 😁

File: code/api/src/modules/product/query.js [hunk 1]
```diff
@@ -43,3 +43,12 @@ export const productTypes = {
   type: new GraphQLList(ProductTypesType),
   resolve: getTypes
 }
+
+// // Products for Survey
```
Comment (nickhartdev) [inline code/api/src/modules/product/query.js:47]: What query will we use for the survey products in the place of this? 

edit: just saw the function down below! Never mind

**Review by nickhartdev — APPROVED** [review]
This looks great, thanks y'all!

**Review by amykarnaze — APPROVED** [review]
Loving the descriptions!

**Pull request merged**